### PR TITLE
Include Weather Underground

### PR DESCRIPTION
**Pull request by kloggy**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_docs/configuration/entity-registry.markdown [hunk 1]
```diff
@@ -55,4 +55,5 @@ value to each entry.
    
 {% linkable_title Supported Components %}
    
-At the moment, only the following components are supported: Z-Wave, Hue, Nest, LIFX, Sonos, Apple TV
+At the moment, only the following components are supported: Z-Wave, Hue, Nest, LIFX, Sonos, Apple TV, Weather Underground 
+Note that for Weather Underground, second and subsequent Personal Weather Station ID (pws_id) with have their monitored conditions suffixed with an index number.
```
Comment (DubhAd) [inline source/_docs/configuration/entity-registry.markdown]: Typo `with have` should be `will have`

Comment (kloggy) [reply source/_docs/configuration/entity-registry.markdown]: Yes, sorry.
I'm new to this.... do I have to change it myself?

Comment (DubhAd) [reply source/_docs/configuration/entity-registry.markdown]: No worries - my own first few PRs were a bit of a mess :)

Yes please.

**Comment by frenck** [comment]
Looks good to me!

Thank you, @kloggy! Great first-time contribution to the documentation!

bitmoji

**Review by frenck — APPROVED** [review]

**Pull request merged**